robot_ws/src/voice_interaction_robot_msgs/msg/AudioData.msg

```diff
@@ -0,0 +1 @@
+uint8[] data
```

Comment:
Once this gets merged we may want to open an issue to remind ourselves to delete this package once the messages are available in apt

Reply:
I don't think there's a need to use the other package, this is the only message type we're using from it and it's super simple.